com.github.donadigo.appeditor

```diff
@@ -0,0 +1,81 @@
+{
+    "app-id": "com.github.donadigo.appeditor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.donadigo.appeditor",
+    "build-options": {
+        "cflags": "-O2",
+        "cxxflags": "-O2"
+    },
+    "cleanup": [
+        "/bin/granite-demo",
```

Comment:
Just personal preference but I like cleanups like this one to be in the module they came from.

Reply:
I have moved the granite specific files to the granite module